sys/include/net/gnrc/lorawan/region.h

```diff
@@ -26,12 +26,18 @@ extern "C" {
 #endif
 
 /**
- * @brief Default LoRaWAN channels for current region (EU868)
+ * @brief Default LoRaWAN channels
  */
 static const uint32_t gnrc_lorawan_default_channels[] = {
+#ifdef LORAWAN_REGION_EU_868
     868100000UL,
     868300000UL,
     868500000UL
+#elif defined(LORAWAN_REGION_IN_865)
+    865062500UL,
+    865402500UL,
+    865985000UL
+#endif
```

Comment:
```suggestion
#if IS_ACTIVE(LORAWAN_REGION_EU_868)
    868100000UL,
    868300000UL,
    868500000UL
#elif IS_ACTIVE(LORAWAN_REGION_IN_865)
    865062500UL,
    865402500UL,
    865985000UL
#endif
```

Reply:
This would only work once I move the macro to CONFIG_ namespace.

Reply:
As commented above, I really think this change should introduce the proper Kconfig file

Reply:
Okay. I shall move them to "CONFIG_" namespace.

Reply:
yes please, and a proper Kconfig file :)

Reply:
Don't you think it's better to do Kconfig exposure on a separate PR ?

Reply:
since there's an ongoing migration to Kconfig, this should at least include the "CONFIG_" prefix. Otherwise there will be an immediate follow up renaming these macros anyway

Reply:
Done